Automattic/woocommerce-payments
# Add QIT integration for running security tests locally

Fixes 155-gh-Automattic/woocommerce-payments-meta

#### Changes proposed in this Pull Request
- Introduce a script to run QIT security tests locally









#### Testing instructions





* Follow the `Setup and running` section in the newly added README.
* **Note** - QIT toolkit has a bug with accepting the zip files that we create in the build release so while that is fixed(hence the draft status), follow these steps:
  * Run the script normally and see the `Invalid zip` error.
  * Unzip the contents, delete the zip and then zip the unzipped folder again.
  * Remove the `npm run build:release` part from the `test:qit` script and run only with the shell script execution part.
* Ensure setup and testing instructions are clear enough. 
-------------------


**Post merge**
